frontend/lib/main.dart

```diff
@@ -134,6 +135,12 @@ class _MyHomePageState extends State<MyHomePage> {
             },
             child: const Text('Go to Password SearchField Page'),
           ),
+          ElevatedButton(
```

Comment:
based on figma design, there is a navbar tab for the admin user list, it can already be separated from this temporary setup